### PR TITLE
added javascript3 week1 homework

### PR DESCRIPTION
**Pull request by krithi-kondana**

### CODE REVIEW

File: javascript/javascript3/week1/mycoolapi/coolapi.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function fetchData() {
+    let pageNumber = document.getElementById('page').value
```
Comment (Rassibassi) [inline javascript/javascript3/week1/mycoolapi/coolapi.js]: why not `const`? what is the pageNumber for?

File: javascript/javascript3/week1/weatherapp/weatherapp.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
+function fetchWeather() {
+  let placeValue = document.getElementById('place').value;
```
Comment (Rassibassi) [inline javascript/javascript3/week1/weatherapp/weatherapp.js]: why not `const`?

File: javascript/javascript3/week1/weatherapp/weatherapp.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  fetch(`https://api.openweathermap.org/data/2.5/weather?q=${placeValue}&appid=84b5c59049054a6076ceb266293c7bc7`)
+    .then(response => response.json())
+    .then(data => {
+      console.log(data);
+      renderWeather(data);
+      renderMaps(data)
+    })
+}
+
+let weather = document.getElementById('btn')
```
Comment (Rassibassi) [inline javascript/javascript3/week1/weatherapp/weatherapp.js]: why not `const`?

File: javascript/javascript3/week1/weatherapp/weatherapp.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                                                 `
+}
+
+let map;
+let marker;
+function renderMaps(data) {
+  let coordinates = {
+    lat: data.coord.lat,
+    lng: data.coord.lon
+  }  //Uncaught (in promise) TypeError: Cannot read property 'coord' of undefined (I dont know how to resolve this.Please help)
```
Comment (Rassibassi) [inline javascript/javascript3/week1/weatherapp/weatherapp.js]: Write a question into slack, then we can help!!!

File: javascript/javascript3/week1/weatherapp/weatherapp.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+
+// my current position optional
+const location1 = document.getElementById('location')
+location1.addEventListener('click', myLocation);
+
+function fetchCurrentPosition(position) {
+  latitude = position.coords.latitude;
```
Comment (Rassibassi) [inline javascript/javascript3/week1/weatherapp/weatherapp.js]: no `let` no `const??

File: javascript/javascript3/week1/giphyapi/gif.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+let errorMessage = document.getElementById('content');
```
Comment (Rassibassi) [inline javascript/javascript3/week1/giphyapi/gif.js]: why not `const`?

File: javascript/javascript3/week1/giphyapi/gif.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+function fetchGif() {
+
+    errorMessage.innerText = '';
+
+    let searchValue = document.getElementById('search').value;
```
Comment (Rassibassi) [inline javascript/javascript3/week1/giphyapi/gif.js]: why not `const`?

Comment (krithi-kondana) [reply javascript/javascript3/week1/giphyapi/gif.js]: Thank you for the review rasmus. I will use const and avoid let hereafter.

**Pull request merged**